.github/workflows/e2e.yaml

```diff
@@ -41,22 +40,47 @@ jobs:
           distribution: "temurin"
           java-version: ${{ matrix.java }}
           cache: "maven"
-      - name: Run Kafka Connect ${{ matrix.kafka }} E2E Tests with ${{ matrix.broker }}
+      - name: Run Kafka Connect ${{ matrix.kafka }} E2E Tests with Apache Kafka
         run: |
-          if [[ ${{ matrix.broker }} == "redpanda" ]]; then 
-            export COMPOSE_FILE=e2e/docker_compose_redpanda.yaml
-          fi
           echo "Running integration tests with Docker Compose file: ${COMPOSE_FILE}"
           docker compose -f ${COMPOSE_FILE} pull
           mvn --batch-mode --no-transfer-progress --errors --update-snapshots clean verify -Drevision=$(git describe --tags --always)
-  # special treatment due to different versioning scheme
+
+  e2e-bitnami:
```

Comment:
question: shall we remove this now that we have the official kafka image? we need to check the contributing guide and docs if they're still referencing bitnami

Reply:
It's up to you. I am fine to keep the Bitnami container.

Reply:
Let's simplify and shorten the tests now that we have an official Kafka image. Good to remove (please check if docs need updates)

Reply:
done